ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,233 @@
+    await page.getByRole('option', { name: 'Ket Format' }).click();
+    await page.getByRole('button', { name: 'Save', exact: true });
+    await openFileAndAddToCanvas('ketfile01t', page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Text tool - Cut/Copy/Paste', async ({ page }) => {
+    // Test case: EPMLSOPKET-2272
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEXT001');
+    await pressButton(page, 'Apply');
+    await page.getByText('TEXT001').click();
+    await selectTopPanelButton(TopPanelButton.Copy, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await page.keyboard.press('Control+v');
+    await page.getByTestId('canvas').click({ position: { x: 250, y: 300 } });
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectNestedTool(page, SelectTool.LASSO_SELECTION);
+    await page.mouse.move(97, 79);
+    await page.mouse.down();
+    await page.mouse.move(943, 114);
+    await page.mouse.move(844, 579);
+    await page.mouse.move(66, 611);
+    await page.mouse.up();
+    await delay(DELAY_IN_SECONDS.TWO);
+    await selectTopPanelButton(TopPanelButton.Cut, page);
+    await page.keyboard.press('Control+v');
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await delay(DELAY_IN_SECONDS.TWO);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await takeEditorScreenshot(page);
+  });
+  test('Text tool - Selection of different types of text objects', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-2274
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEXT');
+    await page.getByRole('dialog').getByRole('textbox').inputValue;
+    await pressButton(page, 'Apply');
+    await page.getByText('TEXT').hover();
+    await page.getByText('TEXT').dblclick();
+    await page.getByRole('dialog').getByRole('textbox').click();
+    await page.keyboard.press('Control+a');
+    await page
+      .getByRole('dialog')
+      .getByRole('textbox')
+      .fill(
+        'PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP',
+      );
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page
+      .getByText(
+        'PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP',
+      )
+      .hover();
+    await page
+      .getByText(
+        'PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP',
+      )
+      .dblclick();
+    await page.keyboard.press('Control+a');
+    await page.keyboard.press('Delete');
+    await pressButton(page, 'Apply');
+    await openFileAndAddToCanvas('longtext_test', page);
+    await selectNestedTool(page, SelectTool.RectangleSelection);
+    await page.mouse.move(97, 79);
```

Comment:
I propose make constants for it.
For example:
const x = 97;
const y = 79;
And then you can use await page.mouse.move(x, y);